src/components/RichText.tsx

```diff
@@ -35,13 +36,22 @@ export function RichText({
     enableTags?: boolean
     authorHandle?: string
     onLinkPress?: LinkProps['onPress']
+    display?: 'color' | 'underlines'
   }) {
   const richText = React.useMemo(
     () =>
       value instanceof RichTextAPI ? value : new RichTextAPI({text: value}),
     [value],
   )
-  const styles = [a.leading_snug, flatten(style)]
+
+  const flattenedStyle = flatten(style)
+  const plainStyles = [a.leading_snug, flattenedStyle]
+  const interactiveStyles = [
+    a.leading_snug,
+    a.pointer_events_auto,
+    display === 'underlines' && a.underline,
+    flattenedStyle,
+  ]
```

Comment:
Wait why is this necessary here 🤔

Reply:
It was an attempt to simplify the code here, there was a lot of repetition of `[...styles, {pointerEvents: 'auto'}]` and I wanted to add `a.underline` to it

Reply:
I can't just pass in underline to the style prop because that would affect all the text

Reply:
Oh of course, I got ya, my b. Kind of a nit, but we have `style`, can we just do `interactivesStyle` or something and pass in styles as an array that get spread here in place of the `display` check? Feels more stable than these other values.

Reply:
I like that better, will do